src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseProvider.cs

```diff
@@ -27,7 +27,7 @@ internal static class IntellisenseProvider
 
         internal static IIntellisense GetIntellisense(PowerFxConfig config)
         {
-            return new Intellisense(config, config.EnumStoreBuilder.Build(), SuggestionHandlers);
```

Comment:
>EnumStoreBuilder [](http://example.com/codeflow?start=51&length=16)

config.EnumStoreBuilder is 
config.SymbolTable.EnumStoreBuilder, which is null. 

Since this is intellisense, it's just reading from the store (doesn't need a full builder). 
That's also important because a reader can compose across multiple symbol tables, whereas a builder can't.